source/publication-server.rst

```diff
@@ -122,6 +122,86 @@ the API.
 However, if you need to use the CLI or API from other machines, then you should
 proxy access to the path '/api' to Krill.
 
+Example nginx configuration
+---------------------------
+
+As introduced above krill has two paths that contain the endpoints. `/api` for the
+krill API that you may want to restrict. `/rfc8181` is used for publication. A
+configuration that allows `192.0.2.0/24` and the IPv6 documentation prefix access to
+the API, and all clients to publish is below.
+
```

Comment:
The example config also has a lot of manually configured SSL options. I don't have an issue with that, but in other examples (might be just in our blogs) we just used the Letsencrypt certbot for this setup. So, in short. Perhaps mention that the example also includes a more elaborate SSL config, but that is of course unrelated to the access restrictions to `/api`